portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,25 +24,68 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
-    ArrayList<String> mylist = new ArrayList<String>();
+    ArrayList<String> commentlist = new ArrayList<String>();
+    ArrayList<String> templist = new ArrayList<String>();
         
-
+/*
     public DataServlet(){
-        mylist.add("Hello this");
-        mylist.add("is a");
-        mylist.add("test");
+        commentlist.add("Hello this");
+        commentlist.add("is a");
+        commentlist.add("test");
     }
-    
+*/
+  
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         
-        String json = new Gson().toJson(mylist);
-
+    
+        //String json = convertToJson(commentlist);
+        String json = new Gson().toJson(templist);
         response.setContentType("application/json;");
-        response.getWriter().println(json);
+        commentlist.add(json);
+
+        for(int i=0; i< commentlist.size(); i++){
+            response.getWriter().println(commentlist.get(i));
+        }
+        
+    }
+
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String text = getParameter(request, "text-input", "");
+
+        // Break the text into individual words.
+        // Name, LastName,Comment
+        String[] words = text.split("\\s*,\\s*");
+        templist.clear();
+        templist.add(words[0]);
+        templist.add(words[1]);
+        templist.add(words[2]);
+
+    }
+
+    private String convertToJson(ArrayList<String> wordspass) {
+        String json = "{";
+        json += "\"Name\": ";
+        json += "\"" + wordspass.get(0) + "\"";
+        json += ", ";
+        json += "\"Last Name\": ";
+        json += "\"" + wordspass.get(1) + "\"";
+        json += ", ";
+        json += "\"Comment\": ";
+        json += wordspass.get(2);
+        json += "}";
+        return json;
+    }
```

Comment:
This isn't used, so should be deleted?

Reply:
Yeah I deleted it but the cloud shell didn't save I guess..